kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/GwtDeviceServiceImpl.java

```diff
@@ -233,20 +234,20 @@ public void startBundle(GwtXSRFToken xsrfToken, String bundleId) throws GwtKuraE
         SystemService systemService = ServiceLocator.getInstance().getService(SystemService.class);
         Bundle[] bundles = systemService.getBundles();
 
-        s_logger.info("Starting bundle with ID: " + bundleId);
+        s_logger.info("Starting bundle with ID: {}", bundleId);
         for (Bundle b : bundles) {
             if (b.getBundleId() == Long.parseLong(bundleId)) {
                 try {
                     b.start();
                     return;
                 } catch (BundleException e) {
-                    s_logger.error("Failed to start bundle {}: {}", new Object[] {b.getBundleId(), e});
+                    s_logger.error("Failed to start bundle " + b.getBundleId(), e);
```

Comment:
Can you please double check. We are using it everyware:
http://slf4j.org/faq.html#paramException